TESTING.md

```diff
@@ -47,6 +47,17 @@ Bugs fixes should include a unit test case which exercises the bug.
 A bug fix may also include new assertions in an existing integration tests for the
 API endpoint.
 
+### Integration tests environment considerations
+
+When adding new tests or modifying existing test under `integration/`, testing environment should be properly considered. `skip.If` from [gotestyourself/skip](https://godoc.org/gotest.tools/skip) can be used to make the test run conditionally. Full testing environment conditions can be found at [environment.go](https://github.com/moby/moby/blob/master/internal/test/environment/environment.go)
```

Comment:
s/`gotestyourself`/`gotest.tools` now 😅 

Also;

- would be good to use a permalink (point to a fixed commit) instead of `master` (in case things move)
- can you wrap this paragraph to 80-characters?

Reply:
Addressed.